plutus-core/executables/uplc/Main.hs

```diff
@@ -268,64 +292,72 @@ runApplyToData (ApplyOptions inputfiles ifmt outp ofmt mode) =
                        Right (d :: Data) ->
                            pure $ UPLC.Program () ver $ mkConstant () d
 
+---------------- Benchmarking ----------------
```

Comment:
I'm not sure if all of the `!`/`$!`/`force`stuff is really necessary (maybe we don't need `!` on `term` and `anonTerm` for example), but at least some of it is and anything extra probably doesn't do any harm.